docs/testing.md

```diff
@@ -283,6 +283,9 @@ const customJestConfig = {
   // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
   // if using TypeScript with a baseUrl set to the root directory then you need the below for alias' to work
   moduleDirectories: ['node_modules', '<rootDir>/'],
+  moduleNameMapper: {
```

Comment:
This is missing a comment above this line explaining why you might add this config.

Reply:
I see. I thought this is done by the comment in line 284. I'll add another more specific one